spec/rubocop/cop/gemspec/ordered_dependencies_spec.rb

```diff
@@ -26,20 +26,16 @@
       end
 
       context 'when gems are not alphabetically sorted' do
-        let(:source) { <<~RUBY }
-          Gem::Specification.new do |spec|
-            spec.#{add_dependency} 'rubocop'
-            spec.#{add_dependency} 'rspec'
-          end
-        RUBY
-
         it 'registers an offense' do
-          expect_offense(offense_message)
-        end
+          expect_offense(<<~RUBY, add_dependency: add_dependency)
+            Gem::Specification.new do |spec|
+              spec.%{add_dependency} 'rubocop'
+              spec.%{add_dependency} 'rspec'
+              ^^^^^^{add_dependency}^^^^^^^^ #{format(message, 'rspec', 'rubocop')}
```

Comment:
Can you replace it with production offense message along with this change?

```diff
--- a/spec/rubocop/cop/gemspec/ordered_dependencies_spec.rb
+++ b/spec/rubocop/cop/gemspec/ordered_dependencies_spec.rb
@@ -7,10 +7,6 @@ RSpec.describe RuboCop::Cop::Gemspec::OrderedDependencies, :config do
     }
   end
   let(:treat_comments_as_group_separators) { false }
-  let(:message) do
-    'Dependencies should be sorted in an alphabetical order within their ' \
-      'section of the gemspec. Dependency `%s` should appear before `%s`.'
-  end

   shared_examples 'ordered dependency' do |add_dependency|
     context "when #{add_dependency}" do
@@ -31,7 +27,7 @@ RSpec.describe RuboCop::Cop::Gemspec::OrderedDependencies, :config do
             Gem::Specification.new do |spec|
               spec.%{add_dependency} 'rubocop'
               spec.%{add_dependency} 'rspec'
-              ^^^^^^{add_dependency}^^^^^^^^ #{format(message, 'rspec', 'rubocop')}
+              ^^^^^^{add_dependency}^^^^^^^^ Dependencies should be sorted in an alphabetical order within their section of the gemspec. Dependency `rspec` should appear before `rubocop`.
             end
           RUBY

@@ -94,7 +90,7 @@ RSpec.describe RuboCop::Cop::Gemspec::OrderedDependencies, :config do
                 # For
                 # test
                 spec.%{add_dependency} 'rspec'
-                ^^^^^^{add_dependency}^^^^^^^^ #{format(message, 'rspec', 'rubocop')}
+                ^^^^^^{add_dependency}^^^^^^^^ Dependencies should be sorted in an alphabetical order within their section of the gemspec. Dependency `rspec` should appear before `rubocop`.
               end
             RUBY
```

Reply:
Sure thing.